.cirrus.yml

```diff
@@ -69,12 +69,11 @@
     PGP_PASSPHRASE: VAULT[development/kv/data/sign data.passphrase]
     #allow deployment of pull request artifacts to repox
     DEPLOY_PULL_REQUEST: true
-  maven_cache:
-    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
+  gradle_cache:
```

Comment:
It will be good to extract Gradle cache configuration as separate config like in SLang: https://github.com/SonarSource/slang/blob/master/.cirrus.yml#L70-L81
Please check if pats for rm are up to date. This is important for upcomming enterprise build